inc/rest-api.php

```diff
@@ -248,42 +253,13 @@ function wp_ai_image_gen_make_api_request($provider, $prompt, $model, $additiona
 
         $body = json_decode(wp_remote_retrieve_body($response), true);
         
-        if (!isset($body['id'])) {
```

Comment:
All this code below was for polling their api to get the image url when it finished generating.